components/provider/src/hello_world.rs

```diff
@@ -0,0 +1,133 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
should this be an `example` rather than a module?

Reply:
I made it a module because:

1. It is extremely useful in unit tests
2. It is useful for unit tests not only in this crate but also in other crates like provider_fs